rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -761,15 +675,11 @@ http {
             client_body_buffer_size                 {{ luaConfigurationRequestBodySize $cfg }};
             proxy_buffering                         off;
 
-            content_by_lua_block {
-              configuration.call()
-            }
+            content_by_lua_file /etc/nginx/lua/nginx/ngx_conf_configuration.lua;
         }
 
         location / {
-            content_by_lua_block {
-                ngx.exit(ngx.HTTP_NOT_FOUND)
-            }
+            content_by_lua_file /etc/nginx/lua/nginx/ngx_not_found.lua;
```

Comment:
Why are we even using Lua here? Can't NGINX itself directly return 404?

Reply:
no idea, at some point I was just copy/pasting the lua scripts to the right location :P let me fix it here